### ISSUE TITLE
Terraform import returning `no matches found:` even though this exists.

### ISSUE THREAD

**Issue by kamal-github** [terraform-google-modules/terraform-google-pubsub#97]
### TL;DR

Terraform import returning `no matches found:` even though this exists.

`terraform plan` output shows the address of the property: `#module.exp_pubsub.google_pubsub_subscription.pull_subscriptions["exp-sub"] will be created`

Below is the import command
`terraform import module.exp_pubsub.google_pubsub_subscription.pull_subscriptions["exp-sub"] "exp-sub"`

### Expected behavior

* has moved to *

### Observed behavior

`terraform import module.exp_pubsub.google_pubsub_subscription.pull_subscriptions["exp-sub"] exp-sub`

zsh: no matches found: module.exp_pubsub.google_pubsub_subscription.pull_subscriptions[exp-sub]

### Terraform Configuration
old configuration

``` hcl
resource "google_pubsub_topic" "topic" {
  name = "exp-topic"

  labels = {
    app = "experimental"
  }
}

resource "google_pubsub_topic" "topic-dead-letter" {
  name = "exp-topic-dead-letter"

  labels = {
    app = "experimental"
  }
}


resource "google_pubsub_subscription" "sub" {
  name = "exp-sub"
  topic = google_pubsub_topic.topic.name

  labels = {
    app = "experimental"
  }

  ack_deadline_seconds = 10                                                 // optional

  dead_letter_policy {
    dead_letter_topic = google_pubsub_topic.topic-dead-letter.id
  }
}
```

#### I want to migrate to managed module, below is the module config

```hcl
module "exp_pubsub" {
  source  = "terraform-google-modules/pubsub/google"
  version = "~> 4.0"

  topic        = "exp-topic"
  project_id = local.project_id

  pull_subscriptions = [
    {
      name                    = "exp-sub"                                               // required
      ack_deadline_seconds    = 10                                                   // optional
      dead_letter_topic       = google_pubsub_topic.topic-dead-letter.id // optional
    }
  ]

  topic_labels  = {
    app = "experimental"
  }

  subscription_labels = {
    app = "experimental"
  }
}
```

### Terraform Version

```sh
4.0.0
```

**Comment by bharathkkb** [comment]
@kamal-github this is likely due to your shell. Try wrapping resource address in quotes.
`terraform import 'module.exp_pubsub.google_pubsub_subscription.pull_subscriptions["exp-sub"]' exp-sub`
